src/data/CustomFormErrorsDefaultRepository.ts

```diff
@@ -0,0 +1,65 @@
+import _ from "lodash";
+import { CustomFormErrorsRepository } from "../domain/validatecustomforms/repositories/CustomFormErrorsRepository";
+import { D2Api } from "../types/d2-api";
+
+export class CustomFormErrorsDefaultRepository implements CustomFormErrorsRepository {
+    constructor(private api: D2Api) {}
+    async get(id: string): Promise<string[]> {
+        const dataSetMetadata: any = await this.api.metadata.d2Api
```

Comment:
`any`. Note that it propagates to the full block, so no type-checking occurs for most of the code.